pycheribuild/projects/cross/compiler-rt.py

```diff
@@ -47,14 +47,14 @@ class BuildCompilerRt(CrossCompileCMakeProject):
     def __init__(self, config: CheriConfig):
         super().__init__(config)
 
-        if self.target_info.is_rtems():
+        if self.target_info.is_rtems() or self.target_info.is_baremetal():
             self.add_cmake_options(CMAKE_TRY_COMPILE_TARGET_TYPE="STATIC_LIBRARY") # RTEMS only needs static libs
             # Get default target (arch) from the triple
             self.add_cmake_options(COMPILER_RT_DEFAULT_TARGET_ARCH=self.target_info.target_triple.split('-')[0])
 
         self.add_cmake_options(
-            LLVM_CONFIG_PATH=BuildCheriLLVM.getBuildDir(self, cross_target=CompilationTargets.NATIVE) / "bin/llvm-config",
-            LLVM_EXTERNAL_LIT=BuildCheriLLVM.getBuildDir(self, cross_target=CompilationTargets.NATIVE) / "bin/llvm-lit",
+            LLVM_CONFIG_PATH=self.sdk_bindir / "llvm-config",
+            LLVM_EXTERNAL_LIT=self.sdk_bindir/ "llvm-lit",
```

Comment:
This is probably better, but the libc++ jenkins job will need adjusting.

Reply:
Is that why the CI check if failing?

Reply:
The failing CI reminds me why I made this change. For the native build there is no sdk_bindir, so it should use it from the build dir. Maybe the best solution is to change it to "build dir by default", but "sdk dir" for Jenkins.